activesupport/test/option_merger_test.rb

```diff
@@ -125,4 +132,8 @@ def method_with_kwargs(*args, **options)
     def method_with_kwargs_only(**options)
       options
     end
+
+    def method_returning_args_and_options(*args, **options)
+      [args, options]
+    end
```

Comment:
Instead of potentially involving kwargs (`**`) here, let's deal with positional args only:

```suggestion
    def method_with_args(*args)
      args
    end
```

And then we can move this method definition right above `method_with_options`.